### PR TITLE
Simplify SimpleWebApp

### PR DESCRIPTION
**Pull request by Shane32**
- Enable implicit usings (default set only)
- Consolidate Program.cs and Startup.cs into a single file using top-level statements
- Use file-scoped namespaces for all files
- Change `langversion` to `latest`

No changes to the code, other than Program.cs and Startup.cs and tests, were made.  No changes at all to GraphQL.Conventions project.

Use 'hide whitespace' in GitHub to reduce the displayed diff

### CODE REVIEW

File: samples/SimpleWebApp/Program.cs [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-    {
-        public static void Main(string[] args)
-        {
-            var host = CreateHostBuilder(args).Build();
+var builder = WebApplication.CreateBuilder(args);
 
-            host.Run();
-        }
+builder.Services.AddSingleton<IBookRepository, BookRepository>();
+builder.Services.AddSingleton<IAuthorRepository, AuthorRepository>();
```
Comment (Shane32) [inline samples/SimpleWebApp/Program.cs:10]: Fixed registrations

File: samples/SimpleWebApp.Tests/EndToEndTests.cs [hunk 2]
```diff
@@ -12,12 +12,17 @@ public async Task Search()
...
         var query = """
             {
-              search (forString:"") {
+              search (forString: "", first: 100) {
                 items {
                   __typename
                   ... on Book {
                     id
                     title
+                    authors {
```
Comment (Shane32) [inline samples/SimpleWebApp.Tests/EndToEndTests.cs:21]: Enhanced test for SampleWebApp here, as previously the test did not test the `[Inject]` bit of code

Comment (Shane32) [reply samples/SimpleWebApp.Tests/EndToEndTests.cs:21]: The mutation added in PR #252 does test `[Inject]` actually.

**Review by tlil — APPROVED** [review]
👍

**Comment by tlil** [comment]
Thanks for all of these fixes and improvements, @Shane32 - much appreciated

**Pull request merged**